doc/running_spire.md

```diff
@@ -0,0 +1,27 @@
+# Options for Running SPIRE
```

Comment:
Since there are many additional ways in which you can run SPIRE, I'm thinking that maybe this could be titled "Start using SPIRE" or something along those lines. Then the description below would be something like "This page describes the some options to get started with SPIRE."
What do you think?

Reply:
Sure, changes seems ok; see the new commit. I changed the file name to match. I also changed the Linux and MacOS sections to remove mentions of executables and source--I think those apply to both really, so those qualifiers aren't needed.